openapi.json

```diff
@@ -863,7 +863,7 @@
             "name": "from_date",
             "type": "string",
             "format": "date",
-            "description": "Retrieve events which occur after and including the provided date. In ISO 8601 format.",
+            "description": "Retrieve events which occur after and including the provided date. In ISO 8601 format. If not set, only upcoming events will be shown.",
```

Comment:
I've removed the default. If nothing is set, the API works as it has up until now: Showing all published events.

If a library has the setting disabled, that automatically unpublishes events, it means the API result will grow over the years. Is that a problem?

Reply:
For now I do not think so.

Since this is a server to server API we should not be that worried about payload size. I would also expect that our caching should help reduce strain.